pkg/graph/schema.resolvers.go

```diff
@@ -43,18 +43,13 @@ func (r *accessibilityRequestResolver) Documents(ctx context.Context, obj *model
 }
 
 func (r *accessibilityRequestResolver) RelevantTestDate(ctx context.Context, obj *models.AccessibilityRequest) (*models.TestDate, error) {
-	allDates := []*models.TestDate{}
-
-	if time.Now().Unix()%3 == 0 {
```

Comment:
What is this modulus checking? Can we put a comment here to the effect?

Reply:
That modulus was generating fake data. But it is being taken out via this PR, so I don't believe a comment would be helpful here.